### DISCUSSION TITLE
Getting error on my sonic pad

### DISCUSSION THREAD

**Discussion by Fannar1984** [OrcaSlicer/OrcaSlicer#3495] [General]
Hello i am getting error on my sonic pad after slice from Orca slicer about extrude below minimum temp but i dont get it when i slice in bambulab or cura or simplify3D can some explein for me what im doing wrong 
regards Fannar
![20240105_212114](https://github.com/SoftFever/OrcaSlicer/assets/15815988/80e2743e-8fca-4546-8381-a7669f4ac190)

**Comment by Fannar1984** [comment]
i find a way to stop getting this error i made this Gcode in orca slicer and all start working :)
G28 ;Home
M109 S220 T0 ; wait for T0 to reach 220 degrees before continuing with any other commands
M190 S50 ; wait until the bed reaches 50 degrees before continuing
G29 ;BED_MESH_PROFILE LOAD="default"
G92 E0 ;Reset Extruder
G1 Z2.0 F3000 ;Move Z Axis up
G1 X2.1 Y20 Z0.28 F5000.0 ;Move to start position
G1 X2.1 Y200.0 Z0.28 F1500.0 E15 ;Draw the first line
G1 X2.4 Y200.0 Z0.28 F5000.0 ;Move to side a little
G1 X2.4 Y20 Z0.28 F1500.0 E30 ;Draw the second line
G92 E0 ;Reset Extruder
G1 Z2.0 F3000 ;Move Z Axis up

**Reply by darco415** [reply]
220 is quite high of temp and it will ooze filament possibly when bedleveling..lower to 180-190
